slack_sdk/oauth/token_rotation/async_rotator.py

```diff
@@ -136,7 +136,7 @@ async def perform_user_token_rotation(  # type: ignore
             refreshed_installation.user_token = refresh_response.get("access_token")
             refreshed_installation.user_refresh_token = refresh_response.get("refresh_token")
             refreshed_installation.user_token_expires_at = int(time()) + int(refresh_response.get("expires_in"))
-            return refreshed_installation  # type: ignore
```

Comment:
This may be still necessary if the pytype validation fails. In the case, could you revert this change?

Reply:
It seems that pytype is happy with this removal. We can keep this as-is,

Reply:
[pytype validation](https://github.com/slackapi/python-slack-sdk/runs/7245594066?check_suite_focus=true) has passed!
Is not right?

Reply:
Sorry I meant you don't need to change this!